src/main/java/jenkins/scm/api/SCMFileSystem.java

```diff
@@ -102,7 +102,11 @@ public static SCMFileSystem of(@NonNull SCM scm) {
     public static SCMFileSystem of(@NonNull SCM scm, @CheckForNull SCMRevision rev) {
         scm.getClass(); // throw NPE if null
         SCMFileSystem fallBack = null;
-        for (Builder b : Jenkins.getInstance().getExtensionList(Builder.class)) {
+        Jenkins j = Jenkins.getInstance();
+        if (j == null) {
+            return fallBack;
```

Comment:
This is just temporary code until I can bump up to 1.572 or newer and simplify. Maybe I should just do that now in `mercurial` and `subversion`; there is not much point in retaining compatibility with 1.568–1.579.